connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -129,22 +129,8 @@ message Fetch {
 // Relation of type [[Aggregate]].
 message Aggregate {
   Relation input = 1;
-
-  // Grouping sets are used in rollups
-  repeated GroupingSet grouping_sets = 2;
-
-  // Measures
-  repeated Measure measures = 3;
-
-  message GroupingSet {
-    repeated Expression aggregate_expressions = 1;
-  }
-
-  message Measure {
-    AggregateFunction function = 1;
-    // Conditional filter for SUM(x FILTER WHERE x < 10)
-    Expression filter = 2;
-  }
+  repeated Expression grouping_expressions = 2;
+  repeated AggregateFunction result_expressions = 3;
```

Comment:
@cloud-fan I still keep this as `AggregateFunction`. `proto.Expression` is a too general type for now.

connect does not have a `NamedExpression`. I will follow up on this to improve this part.

This PR is to improve the grouping_expressions anyway

Reply:
followup improvement SGTM. I don't think we even need `AggregateFunction`. The SQL parser usually just generate `UnresolvedFunction`, and the analyzer will look up the function and figure out if it's scalar/aggregate/window/table value function.